edward/stats/distributions.py

```diff
@@ -9,114 +9,83 @@ class Distribution:
     """Template for all distributions."""
```

Comment:
i remember discussing this with you on slack. i'll create an issue. (for now, let it stay.)